pages/common/csvkit.md

```diff
@@ -6,16 +6,16 @@
 
```

Comment:
I would rephrasing the 3rd link or using `See also:`.

Reply:
Nice catch. I think we can try standardize links to other pages to be `See also:` automatically.